spark-patch/src/main/java/tech/ytsaurus/spyt/patch/SparkPatchAgent.java

```diff
@@ -67,7 +62,7 @@ public static void premain(String args, Instrumentation inst) {
                     .filter(fileName -> fileName.endsWith(".class") && !fileName.startsWith("tech/ytsaurus/"));
 
             Stream<String> externalClasses =
-                    IOUtils.resourceToString("/externalClasses.txt", Charset.defaultCharset()).lines()
+                    resourceToString("/externalClasses.txt").lines()
```

Comment:
```
java.lang.NoSuchMethodError: 'java.lang.String org.apache.commons.io.IOUtils.resourceToString(java.lang.String, java.nio.charset.Charset)'
 at tech.ytsaurus.spyt.patch.SparkPatchAgent.premain(SparkPatchAgent.java:69)
```

Reply:
It seem that `org.apache.commons.io.IOUtils` was not on classpath for java agent but it should because `commons-io-2.11.0.jar` should be on classpath from spark distributive.

Reply:
The problem might be not that it was not on the classpath, but that it was shadowed by an earlier version of this dependency.
I think it is easier to just avoid the dependency usage than to try debugging and fixing it.